octobot_evaluators/api/evaluators.py

```diff
@@ -223,8 +223,10 @@ async def create_all_type_evaluators(config: dict,
         for name, symbol_list in symbols_by_crypto_currencies.items():
             if symbol_list:
                 ticker = get_base_currency(exchange_manager, symbol_list[0])
```

Comment:
I don't understand the `ticker` name

Reply:
ticker is the base of the pair, maybe we should name this variable base ?

Reply:
Ok. We used to name it currency (base sometimes) don't we ? We can also name it ticker but we have something else named ticker in OctoBot.

Reply:
We can't name it currency here since there already is the name of the currency in context (btc for bitcoin for example), let's go for base then?

Reply:
I would prefer base here but if you think ticker is better we can keep it with an additional comment

Reply:
base is better :+1: